dalesbred/src/test/kotlin/org/dalesbred/SqlQueryTest.kt

```diff
@@ -74,5 +78,21 @@ class SqlQueryTest {
         assertEquals(listOf("bar"), query.arguments)
     }
 
+    @Test
+    fun illegalFetchSize() {
+        val query = SqlQuery.query("select * from foo")
+        assertFailsWith<IllegalArgumentException> {
+            query.fetchSize = -1
+        }
+    }
+
+    @Test
+    fun illegalFetchDirection() {
```

Comment:
This is ok, but you can also consider backticks, which allow you to rename the test methods more freely:

```kotlin
@Test
fun `illegal fetch direction`() {
    ...
}
```

I generally use this for new code, but most of Dalesbred's tests have been ported from the old Java code automatically so they retain their old naming.